i18n_translation/i18n_translation.inc

```diff
@@ -3,6 +3,9 @@
 /**
  * @file
  * Internationalization (i18n) module - Translation set.
+ *
+ * @TODO: Remove in 2.x
+ * @codingStandardsIgnoreFile
```

Comment:
We probably have to do this because of all those camelcase nags. On the other hand this lets phpcs totally ignore the file even it's run explicitly on it. Not optimal.